spec/requests/admin/static_slots_spec.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'A StaticSlot request', type: :request do
+  before { sign_in create :member, :super_admin }
+
+  describe 'GET index' do
+    it 'has a successful response' do
+      get admin_static_slots_path
```

Comment:
An empty index page test is near useless, as it should be a standard ActiveAdmin behaviour. You can create a few `StaticSlot` records before the `GET` request to assert that the records do not raise unexpected behaviour